src/components/Common/Button/styles.css

```diff
@@ -3,4 +3,5 @@
     padding: 0.7rem 1.5rem;
     color: var(--white);
     border-radius: 3rem;
+    cursor: pointer;
```

Comment:
few CSS changes req. here : 

1. Fix the width of the buttons.
2. Make the font-weight 600.

Reply:
After completing the code from the video, do cross-check the design once with the original website and the figma designs.